cypress/full/typeahead_page_spec.ts

```diff
@@ -0,0 +1,630 @@
+    });
+
+    it('when user starts to type and if there are no matches found than an error message "No Results Found" is shown',
+      () => {
+        typeahead.clearInputAndSendKeys(noResult, textToInput);
+        typeahead.isDropdownDisabled(noResult);
+        typeahead.isElemTextContain(noResult, typeahead.noResultAlert, alertText);
+      });
+
+    it(`if there are any matches the drop-down is shown. when user clicks on any item then typeahead container
+  auto-fills with a selected State`, () => {
+      typeahead.clearInputAndSendKeys(noResult, stateForCheck);
+      typeahead.clickByText(typeahead.activeDropdown, stateForCheck);
+      typeahead.isDemoContainsTxt(noResult, stateForCheck);
+    });
+  });
+
+  describe('Scrollable', () => {
+    beforeEach('', () => {
+      typeahead.scrollToMenu('Scrollable');
+    });
+
+    const scrollable = typeahead.exampleDemosArr.scrollable;
+    const formTemplate = 'Model: ';
+    const textToInput = 'a';
+    const stateForCheck = 'Michigan';
+    const anotherStateForCheck = 'Idaho';
+
+    it('example contains typeahead input and typeahead card with "Model:"', () => {
+      typeahead.isPreviewExist(scrollable, formTemplate);
+      typeahead.isElementVisible(scrollable, typeahead.inputSelector);
+    });
+
+    it(`when there are any matches then a drop-down with a list of States matches is shown. user is able to scroll
+      down/up to see the matches list`, () => {
+      typeahead.clearInputAndSendKeys(scrollable, textToInput);
+      typeahead.isElementVisible(scrollable, typeahead.activeDropdown);
+      typeahead.isDropdownScrollable(stateForCheck);
+      typeahead.isDropdownScrollable(anotherStateForCheck);
+    });
+
+    it('when user clicks on any item in typeahead drop-down, then typeahead container auto-fills with a selected State',
+      () => {
+        typeahead.clearInputAndSendKeys(scrollable, stateForCheck);
+        typeahead.clickByText(typeahead.activeDropdown, stateForCheck);
+        typeahead.isDemoContainsTxt(scrollable, stateForCheck);
+      });
+  });
+
+  describe('Latinize', () => {
+    beforeEach('', () => {
+      typeahead.scrollToMenu('Latinize');
+    });
+    const latinize = typeahead.exampleDemosArr.latinize;
+    const formTemplate = 'Model: ';
+    const textToInput = 'ú';
+    const nonLatinToInput = 'super';
+    const stateForCheck = 'súper';
+
+    it('example contains typeahead input and typeahead card with "Model:"', () => {
+      typeahead.isPreviewExist(latinize, formTemplate);
+      typeahead.isElementVisible(latinize, typeahead.inputSelector);
+    });
+
+    it('when user starts to type and there are any matches then a drop-down with a list of matches is shown', () => {
+      typeahead.clearInputAndSendKeys(latinize, textToInput);
+      typeahead.isElementVisible(latinize, typeahead.activeDropdown);
+      typeahead.clearInputAndSendKeys(latinize, nonLatinToInput);
```

Comment:
why do you need this last line?

Reply:
added an assertion. it checks that on an ordinary word auto-substituted a latinze word: `super` is equal to `súper`